### DISCUSSION TITLE
New Logo Ideas

### DISCUSSION THREAD

**Discussion by ghost** [Akylas/oss-weather#159] [Ideas]
![Icon #1](https://github.com/Akylas/oss-weather/assets/122128828/ecc393c0-2438-4968-9ae2-601963c14e4e)

**Comment by farfromrefug** [comment]
It looks nice ! If other users want to vote for this please do.
I liked my icon cause it was dead simple and very easy to spot :P

**Comment by ghost** [comment]
> It looks nice ! If other users want to vote for this please do. I liked my icon cause it was dead simple and very easy to spot :P

![Logo #2](https://github.com/Akylas/oss-weather/assets/122128828/4c2d0088-4a02-4444-abdf-a9b411cd409b)

**Comment by farfromrefug** [comment]
@jacobgonzales20 this one i like less because it will be hard to read as a small icon

**Comment by 0-5-0** [comment]
I also like the simplicity of the original yellow circle. It seems like most weather apps use the cloud and sun so the big yellow circle stands out :).

Should this one be updated? What is "NS Weather"? https://github.com/Akylas/oss-weather/blob/master/fastlane/metadata/android/en-US/images/featureGraphic.png

**Comment by farfromrefug** [comment]
@0-5-0 you are right I need a new banner. 
it was the old name (NS for Nativescript)
